package.json

```diff
@@ -188,7 +188,8 @@
     "vscode-extension-tester": "^2.4.0",
     "vscode-test": "^1.2.2",
     "webpack": "^4.41.6",
-    "webpack-cli": "^3.3.11"
+    "webpack-cli": "^3.3.11",
+    "minimist": ">=1.2.2"
```

Comment:
I don't think that this fixes the security warning because that was based on a dependency of a dependency and not a root level one